boba-community/jwt-secret.txt

```diff
@@ -0,0 +1 @@
+688f5d737bad920bdfb2fc2f488d6b6209eebda1dae949a8de91398d932c517a
```

Comment:
I'm guessing this wasn't meant to be checked in? Maybe add to the `.gitignore`?

Reply:
Removed. I thought it's easier for them to use a JWT token set by us and set it as the volume in the docker-compose file. For the security consideration, I removed it and ask users to set it up.